HelloSignalR/Providers/AuthenticationProvider.cs

```diff
@@ -0,0 +1,35 @@
+﻿using AspNet.Security.OpenIdConnect.Server;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Authentication.OpenIdConnect;
+using AspNet.Security.OpenIdConnect.Extensions;
+
+namespace HelloSignalR.Providers
+{
+    public class AuthenticationProvider : OpenIdConnectServerProvider
+    {
+        public override Task ValidateClientAuthentication(ValidateClientAuthenticationContext context)
+        {
+            context.Skipped();
+            return base.ValidateClientAuthentication(context);
```

Comment:
FYI, you don't need to call the base method, unless you also want to call the `OnValidateClientAuthentication` delegate.